src/main/java/fr/lunatech/timekeeper/services/exports/ExportService.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2020 Lunatech S.A.S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.lunatech.timekeeper.services.exports;
+
+import fr.lunatech.timekeeper.csv.ImportedTimeEntry;
+import fr.lunatech.timekeeper.models.imports.UserImportExtension;
+import fr.lunatech.timekeeper.models.time.TimeEntry;
+import fr.lunatech.timekeeper.services.imports.ImportService;
+import fr.lunatech.timekeeper.timeutils.TimeKeeperDateUtils;
+import org.apache.camel.Exchange;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.transaction.Transactional;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@ApplicationScoped
+public class ExportService {
+
+    private static Logger logger = LoggerFactory.getLogger(ImportService.class);
+
+    /**
+     * get importedTimeEntries between two date from database
+     * @param exchange that contains an header with the dates
+     * @return a List of TimeEntry
+     */
+    @Transactional
+    public List<ImportedTimeEntry> getImportedTimeEntriesBetweenTwoDate(Exchange exchange) {
+        var start = (LocalDate) (exchange.getIn().getHeader("startDate"));
+        var end = (LocalDate) (exchange.getIn().getHeader("endDate"));
+
+        final LocalDateTime startDateTime = start.atTime(0, 1);
+        final LocalDateTime endDateTime = end.atTime(23, 59);
+
+        try (final Stream<TimeEntry> timeEntryStream = TimeEntry.streamAll()) {
+            logger.debug("StreamAll timeentries succeded, now filtering");
+            return timeEntryStream
+                    .filter(timeEntry -> (timeEntry.startDateTime.isAfter(startDateTime) || timeEntry.startDateTime.isEqual(startDateTime))
+                            && (timeEntry.endDateTime.isBefore(endDateTime) || timeEntry.endDateTime.isEqual(endDateTime))
+                    )
+                    .map(this::computeImportedTimeEntry)
+                    .collect(Collectors.toList());
+        } catch (Exception e) {
+            logger.warn("Exception -> " + e);
```

Comment:
Sonar would says : "Format specifiers should be used instead of string concatenation."
`logger.warn("Exception -> {}" , e);` 
It's minor.